users/src/components/App.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import Loader from 'react-loader-spinner';
+import {
+  Form, FormGroup, Label, Input, Button,
+  ListGroup, ListGroupItem, ListGroupItemHeading, ListGroupItemText
+} from 'reactstrap';
+import axios from 'axios';
+import './App.css';
+
+const serverHandshake = () => axios.create({
+  baseURL: 'http://penguin.linux.test:4000/api'
+});
```

Comment:
it's good that you're staying comfortable with the axios.create method, just be aware that it's not _super_ necessary unless you also need to set a timeout for something or adding something to your header. otherwise you could just store the baseURL in a variable and just reference it in all your axios calls and skip this step